src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultDocumentVersionCache.cs

```diff
@@ -28,14 +28,14 @@ public DefaultDocumentVersionCache(ForegroundDispatcher foregroundDispatcher)
             _documentLookup = new Dictionary<string, List<DocumentEntry>>(FilePathComparer.Instance);
         }
 
-        public override void TrackDocumentVersion(DocumentSnapshot documentSnapshot, long version)
+        public override void TrackDocumentVersion(DocumentSnapshot documentSnapshot, int? version)
         {
             if (documentSnapshot == null)
             {
                 throw new ArgumentNullException(nameof(documentSnapshot));
             }
 
-            if (version < 0)
+            if (version is null)
```

Comment:
We basically already had the null version handling, it was just using -1, so I converted anywhere that handled -1 to use int?